docs/tasks/administer-cluster/highly-available-master.md

```diff
@@ -92,11 +92,11 @@ $ KUBE_GCE_ZONE=replica-zone KUBE_REPLICATE_EXISTING_MASTER=true ./cluster/kube-
 
 ## Best practices for replicating masters for HA clusters
 
-* Try to place masters replicas in different zones. During a zone failure, all master placed inside the zone will fail.
+* Try to place masters replicas in different zones. During a zone failure, all masters placed inside the zone will fail.
```

Comment:
can you also change "masters replicas" to "master replicas" while we're here? Thanks!